src/nycdb/transform.py

```diff
@@ -112,7 +112,7 @@ def with_geo(table):
     for row in table:
         try:
             coords = (float(row['xcoord']), float(row['ycoord']))
-            lng, lat = ny_state_coords_to_lat_lng(*coords)
+            lat, lng = ny_state_coords_to_lat_lng(*coords)
```

Comment:
Heh, the naming of this function actually makes more sense now that it's returning a tuple in the same order as the function name!